### PR TITLE
[Improvement][Deployment] Use minio as defaut storage if deployed in k8s

### PR DESCRIPTION
**Pull request by EricGao888**
## Purpose of the pull request

* Set `minio` as DS default storage when deployed in K8S, in this way, users could use resource center out-of-the-box.

## Brief change log

* Already described above.

## Verify this pull request

* Verified manually.

### CODE REVIEW

**Comment by EricGao888** [comment]
Seems the version and app version of DS helm chart is still `3.1.0`. Should we change it to `3.1.3`? cc @zhongjiajie 

https://github.com/apache/dolphinscheduler/blob/dd0d73036ed5065aa4f4920700aaa0aabc13ef7e/deploy/kubernetes/dolphinscheduler/Chart.yaml#L34-L43

**Review by kezhenxu94 — DISMISSED** [review]

**Comment by zhongjiajie** [comment]
> Seems the version and app version of DS helm chart is still `3.1.0`. Should we change it to `3.1.3`? cc @zhongjiajie
> 
> https://github.com/apache/dolphinscheduler/blob/dd0d73036ed5065aa4f4920700aaa0aabc13ef7e/deploy/kubernetes/dolphinscheduler/Chart.yaml#L34-L43

good catch, I think we should change to the latest version, could you please also change our release docs to tell release manager change it during the release process?

**Comment by EricGao888** [comment]
> > Seems the version and app version of DS helm chart is still `3.1.0`. Should we change it to `3.1.3`? cc @zhongjiajie
> > https://github.com/apache/dolphinscheduler/blob/dd0d73036ed5065aa4f4920700aaa0aabc13ef7e/deploy/kubernetes/dolphinscheduler/Chart.yaml#L34-L43
> 
> good catch, I think we should change to the latest version, could you please also change our release docs to tell release manager change it during the release process?

Sure, will do : )

**Comment by zhongjiajie** [comment]
BTW, find flake dead link https://codecov.io/gh/apache/dolphinscheduler/branch/dev, could you also change url to https://app.codecov.io/gh/apache/dolphinscheduler/tree/dev?

**Comment by EricGao888** [comment]
> BTW, find flake dead link https://codecov.io/gh/apache/dolphinscheduler/branch/dev, could you also change url to https://app.codecov.io/gh/apache/dolphinscheduler/tree/dev?

Sure, will fix it in this PR.

**Comment by kezhenxu94** [comment]
This reminds me we should also update the release doc to help release manager to also publish the helm chart so that users don't need to download the source codes in order to deploy with Helm Chart. FYI the helm chart repo is here https://hub.docker.com/r/apache/dolphinscheduler-helm/tags


Let me write up the release doc to include guide how to release the Helm Chart.

**Comment by zhongjiajie** [comment]
> This reminds me we should also update the release doc to help release manager to also publish the helm chart so that users don't need to download the source codes in order to deploy with Helm Chart. FYI the helm chart repo is here https://hub.docker.com/r/apache/dolphinscheduler-helm/tags
> 
> Let me write up the release doc to include guide how to release the Helm Chart.

is there any guide about how to release helm chart to docker hub?

**Comment by kezhenxu94** [comment]
> > This reminds me we should also update the release doc to help release manager to also publish the helm chart so that users don't need to download the source codes in order to deploy with Helm Chart. FYI the helm chart repo is here https://hub.docker.com/r/apache/dolphinscheduler-helm/tags
> > Let me write up the release doc to include guide how to release the Helm Chart.
> 
> is there any guide about how to release helm chart to docker hub?

Check this https://github.com/apache/dolphinscheduler/pull/13385

File: deploy/kubernetes/dolphinscheduler/Chart.yaml [hunk 1]
```diff
@@ -35,11 +35,11 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 3.1.0
+version: latest
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
-appVersion: 3.1.0
+appVersion: latest
```
Comment (zhongjiajie) [inline deploy/kubernetes/dolphinscheduler/Chart.yaml:42]: I am not sure whether the latest tag work here or not, I think there are some cases chart.yaml in dev branch not work in our latest version?  @kezhenxu94 do you have time to take a look?

**Review by zhongjiajie — COMMENTED** [review]
others LGTM

**Pull request closed without merging**